plugins.txt

```diff
@@ -51,7 +51,7 @@ commons-text-api:1.10.0-78.v3e7b_ea_d5a_fe1
 config-file-provider:959.vcff671a_4518b_
 configuration-as-code:1714.v09593e830cfa
 copyartifact:722.v0662a_9b_e22a_c
-credentials:1293.vff276f713473
+credentials:1307.v3757c78f17c3
```

Comment:
- https://plugins.jenkins.io/credentials/releases/#version_1304.v5ec13eecef46
- https://plugins.jenkins.io/credentials/releases/#version_1305.v04f5ec1f3743
- https://plugins.jenkins.io/credentials/releases/#version_1307.v3757c78f17c3

=> bug fixes and dependencies